### PR TITLE
added maxline parameter to Multicell function

### PR DESCRIPTION
**Pull request by rserale**
When calling Multicell function, if the number of lines (implicit or
explicit) reaches the limit specified by maxline parameter, the
function stops and returns the non-printed part.

If maxline equals 0, there will be no limitation of the number of lines.

### CODE REVIEW

**Comment by kazsaj** [comment]
Hello @rserale 

The changes look good, they should not affect existing functionality, but would you be able to re-format the changes to adhere to the projects internal formatting? I mean using three char prefixes for indicating the variable type and using space after if just before the open bracket (`if (` instead of `if(`).

I can try to these changes myself and push them to your master branch if your happy with that (and if github's permission system will allow me to do it).

Thanks,
Kaz

**Comment by rserale** [comment]
Hello @kazsaj 

I will do this on this week :)

**Comment by kazsaj** [comment]
Thanks!

**Comment by rserale** [comment]
Hello @kazsaj,

I reformatted my code syntax as you asked to suit with the project's internal formatting.
Is it okay like this ?

File: src/tFPDF/PDF.php [hunk 1]
```diff
@@ -1431,8 +1431,10 @@ public function Cell($flt_width, $flt_height = 0, $str_text = '', $int_border =
      * @param int $int_border
      * @param string $str_alignment
      * @param bool $bol_fill
+     *
```
Comment (kazsaj) [inline src/tFPDF/PDF.php:1435]: Thanks for reformatting. Now for the last request, I promise!

Could you please add here
```
@param int int_maxline
```

And I will be happy to merge the changes.

Comment (rserale) [reply src/tFPDF/PDF.php:1435]: Done ! :)

**Comment by kazsaj** [comment]
Hi Remi,

I just created a new release of the library ( https://github.com/DocnetUK/tfpdf/releases/tag/v2.0.6 ), which includes your changes to the MultiCell method.

**Comment by rserale** [comment]
Thank you ! I hope it will help. We needed this so bad in my company :)

**Pull request merged**